src/modules/ChannelSettings/components/ChannelSettingsUI/hooks/useMenuItems.tsx

```diff
@@ -57,7 +57,7 @@ interface UseMenuItemsParams {
   renderUserListItem?: (props: UserListItemProps) => ReactNode;
 }
 export const useMenuItems = (props?: UseMenuItemsParams): MenuItems => {
-  const { renderUserListItem } = props;
+  const { renderUserListItem = undefined } = props ?? {};
```

Comment:
```suggestion
  const renderUserListItem = props?.renderUserListItem;
```
We can achieve the same with this one line too.